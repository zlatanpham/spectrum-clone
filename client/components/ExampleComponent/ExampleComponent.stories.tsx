import { storiesOf } from '@storybook/react';
import React from 'react';
import ExampleComponent from '.';

storiesOf('components/ExampleComponent', module).add('Basic', () => (
  <ExampleComponent>Test deploy</ExampleComponent>
));
