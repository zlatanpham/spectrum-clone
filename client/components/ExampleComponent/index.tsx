import React from 'react';
import tw from 'tailwind.macro';
import styled from 'styled-components';

const Container = styled.div`
  ${tw`p-8 bg-gray-300 text-2xl font-bold`}
`;

interface ExampleComponentProps {}

const ExampleComponent: React.FC<ExampleComponentProps> = ({ children }) => {
  return <Container className="border border-gray m-5">{children}</Container>;
};

export default ExampleComponent;
