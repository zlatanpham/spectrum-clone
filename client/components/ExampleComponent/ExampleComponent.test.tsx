import React from 'react';
import { render } from '@testing-library/react';
import ExampleComponent from '.';

describe('ExampleComponent', () => {
  it('should match snapshot', () => {
    const { container } = render(<ExampleComponent />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
