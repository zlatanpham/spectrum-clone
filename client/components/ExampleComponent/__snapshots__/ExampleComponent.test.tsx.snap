// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExampleComponent should match snapshot 1`] = `
.c0 {
  padding: 2rem;
  background-color: #EBECED;
  font-size: 1.5rem;
  font-weight: 700;
}

<div
  class="c0 border border-grey m-5"
/>
`;
